test/e2e/extensions/high_availability_test.go

```diff
@@ -41,16 +41,22 @@ func TestGameServerHealthyAfterDeletingPodWhileOneExtensionsDown(t *testing.T) {
 
 	list, err := getAgoneseExtensionsPods(ctx)
 	logger.Infof("Length of pod list is %v", len(list.Items))
+	for i := range list.Items {
+		logger.Infof("Name of extensions pod %v: %v", i, list.Items[i].ObjectMeta.Name)
+		logger.Infof("Host IP %v", list.Items[i].Status.HostIP)
+		logger.Infof("Pod IPs %v", list.Items[i].Status.PodIPs)
+	}
 	assert.NoError(t, err, "Could not get list of Extension pods")
 	assert.Greater(t, len(list.Items), 1, "Cluster has no Extensions pod or has only 1 extensions pod")
 
-	logger.Info("Removing one of the Extensions Pods")
+	logger.Infof("Removing one of the Extensions Pods: %v", list.Items[1].ObjectMeta.Name)
 	deleteAgonesExtensionsPods(ctx, t)
+	time.Sleep(5 * time.Second)
 
 	assert.NoError(t, waitForAgonesExtensionsRunning(ctx))
 
-	logger.Info("Creating default game server")
 	gs := framework.DefaultGameServer(defaultNs)
+	logger.Infof("Creating game-server %s...", gs.Name)
 	readyGs, err := framework.CreateGameServerAndWaitUntilReady(t, defaultNs, gs)
 	assert.NoError(t, err, "Could not get a GameServer ready")
```

Comment:
```suggestion
	require.NoError(t, err, "Could not get a GameServer ready")
```

Just trying to protect against the panics, which will also make debug easier.

Reply:
Done